workspace/src/app/views/shared/modals/CreateArtefactModal/ArtefactForms/ParameterAutoCompletion.tsx

```diff
@@ -187,6 +221,7 @@ export const ParameterAutoCompletion = ({
                 readOnly: !!readOnly,
                 ...inputProps,
             }}
+            loadMoreResults={fetchMore ? loadMoreResults : undefined}
```

Comment:
Would it be better that the `loadMoreResults` function signals the caller via its return value, e.g. `undefined` that there are no more results, instead of changing props? Then the decision to stop firing request could be moved into the `SuggestFiled` component.